examples/index.html

```diff
@@ -103,15 +121,15 @@ <h1>Image Sequencer</h1>
       var sequencer;
     })
 
+
     function create_gif(){
```

Comment:
I also wonder if the gif-related code could be moved into a file like some of our other UI code:

```html
   <script src="lib/defaultHtmlStepUi.js" charset="utf-8"></script>
```

It could be in `lib/gif.js` perhaps? That would help keep the demo code tidier!